src/client/elm/TimeHelpers.elm

```diff
@@ -16,8 +16,6 @@ formatTime zone posix =
         ++ (String.padLeft 2 '0' <| String.fromInt <| Time.toMinute zone posix)
         ++ ":"
         ++ (String.padLeft 2 '0' <| String.fromInt <| Time.toSecond zone posix)
-        ++ "."
-        ++ (String.padLeft 3 '0' <| String.fromInt <| Time.toMillis zone posix)
```

Comment:
I seem to remember that we explicitly added the milliseconds because otherwise it sometimes looked like two blocks on the same chain were made at the same time.

Reply:
I won't insist on it but I don't think that those milliseconds provide a lot of value even in that case. 
- The blocks still have different hashes and potentially different contents.
- If someone really needs to know what block came first they can use the client to get the timestamps of the blocks.
- In my opinion, it leads to too much information in that one line, and it makes it harder to read the time.
- It currently leads to overflow on the line with the timestamp.
![Screenshot from 2021-05-10 10-06-10](https://user-images.githubusercontent.com/394853/117626507-75c4fa00-b177-11eb-85b5-bb524f579bbb.png)

Reply:
I don't want to argue here, but in the past it was without milliseconds. Some found it confusing we have two blocks at the same time, and requested this to be changed. Thus we cannot go back since this would constitute deliberately introducing something we were asked to fix.

With the current parameters the last digit will always be `0`, if that helps.

Reply:
OK, fair enough, brought back milliseconds and removed last digit.